apps/researcher/src/lib/get-format-date.ts

```diff
@@ -0,0 +1,31 @@
+'use server';
+
+import {getLocale, getTranslations} from 'next-intl/server';
+
+export async function getDateFormatter() {
```

Comment:
If I understand this function correctly it does two things:

1. Create an instance of a date formatter (`formatter`)
2. Format a date (`formatDate`)

The function name, to me, suggests that it returns the instance of (1), but it actually returns a function that does (2)? If so, is it possible to rename the function to make this clearer? For example: `formatDate(date: Date)`?